tiledb/sm/array_schema/array_schema.cc

```diff
@@ -274,14 +274,13 @@ const FilterPipeline* ArraySchema::coords_filters() const {
 
 Compressor ArraySchema::coords_compression() const {
   auto compressor = coords_filters_.get_filter<CompressionFilter>();
-  assert(compressor != nullptr);
-  return compressor->compressor();
+  return (compressor == nullptr) ? Compressor::NO_COMPRESSION :
```

Comment:
Is this just cleaning up the assert or was there some substantive change here?

Reply:
There was a bug I caught accidentally when dumping the array schema of an array with no compression on coordinates.

Reply:
I have edited the PR description for clarity.